code/modules/crafting/stack_recipes/recipes_planks.dm

```diff
@@ -39,10 +39,21 @@
 	result_type            = /obj/item/chems/glass/bucket/wood
 	difficulty             = MAT_VALUE_EASY_DIY
 
+/decl/stack_recipe/planks/bolt
+	result_type            = /obj/item/stack/material/bow_ammo/bolt
+	difficulty             = MAT_VALUE_EASY_DIY
+
+/decl/stack_recipe/planks/arrow
+	result_type            = /obj/item/stack/material/bow_ammo/arrow
+	difficulty             = MAT_VALUE_HARD_DIY
+
 /decl/stack_recipe/planks/bow
 	result_type            = /obj/item/gun/launcher/bow/crafted
 	difficulty             = MAT_VALUE_VERY_HARD_DIY
 
+/decl/stack_recipe/planks/bow/fancy
+	result_type            = /obj/item/gun/launcher/bow/fancy/crafted
```

Comment:
this makes it impossible to craft the base bow type because the name is identical. `title = "engraved bow"` maybe?